compiler-core/src/erlang/tests/snapshots/gleam_core__erlang__tests__case__spread_empty_list.snap

```diff
@@ -0,0 +1,15 @@
+---
+source: compiler-core/src/erlang/tests/case.rs
+expression: "\npub fn main() {\n  case [] {\n    [..] -> 1\n  }\n}\n"
+---
+-module(my@mod).
+-compile([no_auto_import, nowarn_unused_vars, nowarn_unused_function, nowarn_nomatch]).
+
+-export([main/0]).
+
+-spec main() -> integer().
+main() ->
+    case [] of
+        [_] ->
```

Comment:
Sorry about the late comment.

This matches a list with exactly one element, not a list with any tail.

I was wondering if it would be sufficient to just return the tail pattern:

```rust
    let elems = if let Some(final_tail) = tail {
        if elems.is_empty() {
            // must not surround with [], but
            // does this need .nest() or .group()?
            return final_tail;
        }
        elems.append(break_(" |", " | ")).append(final_tail)
```